DownFall/Scenes/GameScene.swift

```diff
@@ -265,6 +199,12 @@ extension GameScene {
         self.renderer?.touchesBegan(touches, with: event)
     }
     
+    override func touchesCancelled(_ touches: Set<UITouch>, with event: UIEvent?) {
+        // avoid inputing touchEnded when a touch is cancelled.
+        // FIXME: there is probably a better way to fix this
+        touchWasSwipe = true
+    }
```

Comment:
Let's just create another bool for touchWasCancelled.  Check it in touchesEnded and then reset it to the default value